app/views/_user_util_links.html.erb

```diff
@@ -34,6 +34,11 @@ Unless required by applicable law or agreed to in writing, software distributed
       <% end %>
     </li>
     <% end %>
+    <% if user_signed_in? %>
+    <li>
+      <%= link_to 'Playlists', playlists_path, id:'playlists_nav' %> 
```

Comment:
I think we also need the conditional class that does the background highlight like the other global nav tabs:

<li class=<%= active_for_controller('playlists') %>><%= link_to 'Playlists', playlists_path, id: 'playlists_nav' %></li>